sklearn/discriminant_analysis.py

```diff
@@ -1034,12 +1108,9 @@ def predict_proba(self, X):
         C : ndarray of shape (n_samples, n_classes)
             Posterior probabilities of classification per class.
         """
-        values = self._decision_function(X)
         # compute the likelihood of the underlying gaussian models
         # up to a multiplicative constant.
-        likelihood = np.exp(values - values.max(axis=1)[:, np.newaxis])
-        # compute posterior probabilities
-        return likelihood / likelihood.sum(axis=1)[:, np.newaxis]
+        return super().predict_proba(X)
```

Comment:
I don't think we need these methods that they only call their parent's method. Unless there's something specific in their docstring.

Reply:
QDA does not support sparse matrix, so we'll need to override superclass' docstrings. I think that's the only difference.